docs/design/lexical_conventions/symbolic_tokens.md

```diff
@@ -77,103 +63,35 @@ overloading.
-whitespace between the operator and its operand.
-
-But in some cases omitting the whitespace around a binary operator aids
-readability, such as in expressions like `2*x*x + 3*x + 1`, hence we have an
-allowance in such cases. In this case the operator with whitespace on neither
-side, if the token immediately before the operator indicates it is the end of an
-operand, and the token immediately after the operator indicates it is the
-beginning of an operand, the operator is treated as binary.
-
-The defined set of tokens that constitutes the beginning or end of an operand
-are:
-
--   Identifiers, as in `x*x + y*y`.
--   Literals, as in `3*x + 4*y` or `"foo"+s`.
--   Brackets of any kind, facing away from the operator, as in `f()*(n + 3)` or
-    `args[3]*{.real=4, .imag=1}`.
-
-For error recovery purposes, this rule functions best if no expression context
-can be preceded by a token that looks like the end of an operand and no
-expression context can be followed by a token that looks like the start of an
-operand. One known exception to this is in function definitions:
-
-```
-fn F(p: Int *) -> Int * { return p; }
-```
-
-Both occurrences of `Int *` here are erroneous. The first is easy to detect and
-diagnose, but the second is more challenging, if `{...}` is a valid expression
-form. We expect to be able to easily distinguish between code blocks starting
-with `{` and expressions starting with `{` for all cases other than `{}`.
-However, the code block `{}` is not a reasonable body for a function with a
-return type, so we expect errors involving a combination of misplaced whitespace
-and `{}` to be rare, and we should be able to recover well from the remaining
-cases.
-
-From the perspective of token formation, the whitespace rule means that there
-are four _variants_ of each symbolic token:
-
--   A symbolic token with whitespace on both sides is a _binary_ variant of the
-    token.
--   A symbolic token with whitespace on neither side, where the preceding token
-    is an identifier, literal, or closing bracket, and the following token is an
-    identifier, literal, or `(`, is also a _binary_ variant of the token.
--   A symbolic token with whitespace on neither side that does not satisfy the
-    preceding rule is a _unary_ variant of the token.
--   A symbolic token with whitespace on the left side only is a _prefix_ variant
-    of the token.
--   A symbolic token with whitespace on the right side only is a _postfix_
-    variant of the token.
-
-When used in non-operator contexts, any variant of a symbolic token is
-acceptable. When used in operator contexts, only a binary variant of a token can
-be used as a binary operator, only a prefix or unary variant of a token can be
-used as a prefix operator, and only a postfix or unary variant of a token can be
-used as a postfix operator.
+| Symbolic Tokens | Explanation                                                                                                |
```

Comment:
Doesn't look like most of the assignment operators from proposal #2511 have been included. Is that intentional?

Reply:
@josh11b Actually I added the operators according to [this](https://github.com/carbon-language/carbon-lang/pull/2657#discussion_r1137826711). So shall I add the ones in #2511 too???

Reply:
Or add a note that the ones in #2511 are still to be added.